opendg/_storage/backends/__init__.py

```diff
@@ -0,0 +1,7 @@
+from opendg._storage.backends.dict_backend import DGStorageDictBackend
+
+DGStorageBackends = {
+    'DictionaryBackend': DGStorageDictBackend,
```

Comment:
Perhaps using the class name string itself is a bit better, I'm not entirely sure